mautic/mautic
# Changed willdurand/oauth-server-bundle to use our fork with parameter encoding fix

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | #1323
| BC breaks? | n
| Deprecations? | n

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:

The oauth1a server we're using has a bug that does not properly comply with https://tools.ietf.org/html/rfc5849#section-3.4.1.3.2. The upstream vendor doesn't seem to support the dependency any longer so it's been forked to Mautic's org, fixed the issue (thanks @cedel1), and Mautic's composer set to use the fork for the vendor. 

#### Steps to test this PR:
1. composer install
2. Using  a 3rd party API tester, test the endpoint `contacts?search=foo@bar.com`, it should not get the signature_invalid error (the API Tester included in Mautic's API tester cannot be used for this till it is fixed to comply with the same bug - https://github.com/mautic/api-library/pull/54)

### As applicable
#### Steps to reproduce the bug:
1. Repeat above before applying the PR - should get signature_invalid
